src/transaction/log_replication_atomic.cpp

```diff
@@ -216,6 +217,28 @@ namespace cublog
 	      }
 	    break;
 	  }
+	  case LOG_SCHEMA_MODIFICATION_LOCK:
+	  {
+	    char *classname = NULL;
+
+	    m_redo_context.m_reader.advance_when_does_not_fit (sizeof (LOG_REC_SCHEMA_MODIFICATION_LOCK));
+	    const LOG_REC_SCHEMA_MODIFICATION_LOCK log_rec =
+		    m_redo_context.m_reader.reinterpret_copy_and_add_align<LOG_REC_SCHEMA_MODIFICATION_LOCK> ();
+
+	    /* TODO:
```

Comment:
Is going to be removed at the end of the epic? Otherwise, it should be wrapped with the DEBUG definition.